torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2827,11 +2876,30 @@ def get_symbol(tensor_dim_src):
                 concrete_val = self.evaluate_expr(sympy.Eq(s1, s2))
                 if not concrete_val:
                     raise ConstraintViolationError(
-                        f"{src1.name()} = {self.var_to_val[s1]}"
+                        f"{src1.name()} = {s1.subs(self.var_to_val)}"
```

Comment:
Odd, is s1 not guaranteed to be a symbol here? Maybe we should rename them (and get_symbol?)

Reply:
Right, we can no longer assume they are symbols. A placeholder dim might be a symint that holds an expression, something like `s0 + 1`. (It might have started as `s1` but gotten replaced away.)

Reply:
I'll rename get_symbol for this to be clear.